Quantum-Control-Applications/Superconducting/Two-Fixed-Coupled-Transmons/18e_CR_calib_cr_cancel_amplitude.py

```diff
@@ -1,13 +1,17 @@
 """
-                                 CR_calib_cancel_drive_amplitude
+        CR_calib_cancel_drive_amplitude
```

Comment:
The purpose of this one is to fins the cancel pulse amplitude no?

Reply:
As replied in the comment at 18d, I simply this.